src/components/pages/dashboard/ManageDomain/cmp.tsx

```diff
@@ -42,6 +42,7 @@ export default function ManageDomain() {
 
   const name = domain.name || ellipseAddress(domain.id)
   const typeName = EntityTypeName[domain.type]
+  const is_confidential = domain.target == EntityDomainType.Confidential
```

Comment:
in javascript by convention variables are camelCase, please replace it everywhere it has been set in snake_case